src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.kt

```diff
@@ -77,28 +75,33 @@ class SonarLintRulePanel(private val project: Project) : JBLoadingPanel(BorderLa
     private val paramsPanel = JBPanel<JBPanel<*>>(GridBagLayout())
     private val securityHotspotHeaderMessage = JEditorPane()
     private val ruleDetailsLoader = RuleDetailsLoader()
+    private var finding: Finding? = null
+    private var ruleDetails: ActiveRuleDetailsDto? = null
 
     init {
         add(JBPanel<JBPanel<*>>(BorderLayout()).apply {
-            ruleNameLabel.font = UIUtil.getLabelFont().deriveFont((UIUtil.getLabelFont().size2D + JBUIScale.scale(3))).deriveFont(Font.BOLD)
-            add(ruleNameLabel, BorderLayout.NORTH)
+            add(ruleNameLabel.apply {
+                font = UIUtil.getLabelFont().deriveFont((UIUtil.getLabelFont().size2D + JBUIScale.scale(3))).deriveFont(Font.BOLD)
+            }, BorderLayout.NORTH)
             add(headerPanel, BorderLayout.CENTER)
             add(securityHotspotHeaderMessage.apply {
                 contentType = UIUtil.HTML_MIME
                 (caret as DefaultCaret).updatePolicy = DefaultCaret.NEVER_UPDATE
                 editorKit = UIUtil.getHTMLEditorKit()
-                border = JBUI.Borders.empty(10)
+                border = JBUI.Borders.empty()
                 isEditable = false
                 isOpaque = false
+                isFocusable = false
 
                 addHyperlinkListener(BrowserHyperlinkListener.INSTANCE)
-            }, BorderLayout.PAGE_END)
+            }, BorderLayout.SOUTH)
```

Comment:
Based on [Oracle documentation](https://docs.oracle.com/javase/tutorial/uiswing/layout/border.html), we should keep usage of `PAGE_END` IMO

`Before JDK release 1.4, the preferred names for the various areas were different, ranging from points of the compass (for example, BorderLayout.NORTH for the top area) to wordier versions of the constants we use in our examples. The constants our examples use are preferred because they are standard and enable programs to adjust to languages that have different orientations.`

Reply:
Interesting, but I am not sure this is what we want. In the case of languages with inverted orientation, I am not sure we want to have our layout inverted.
See https://stackoverflow.com/a/62894110/534773
I suggest keeping using NORTH/SOUTH EAST/WEST at the moment.